### PR TITLE
feat(syntax): better syntax highlight

### PR DESCRIPTION
**Pull request by jukben**
~I didn't see a reason for the logic inside of highlight method so I've forced "JSX" language.~

@satya164 Are you willing to bump and publish a new version?

### CODE REVIEW

File: src/templates/Markdown.js [hunk 1]
```diff
@@ -2,84 +2,18 @@
...
-    try {
-      return hljs.highlightAuto(text).value;
-    } catch (err) {
-      // Do nothing
-    }
-  }
-
-  return '';
-}
+const illuminate = require('illuminate-js');
```
Comment (satya164) [inline src/templates/Markdown.js]: Can you use ES imports here?

File: src/templates/Markdown.js [hunk 2]
```diff
@@ -2,84 +2,18 @@
...
   const { source, className, ...rest } = props;
   return (
-    <div {...rest} className={`${markdown} ${className}`}>
+    <div {...rest} className={`${className}`}>
       <Remarkable
         source={source}
         options={{
           linkify: true,
-          highlight,
+          highlight: text => illuminate.highlight(text, 'jsx'),
```
Comment (satya164) [inline src/templates/Markdown.js]: We don't want to force `jsx`. Highlighter should respect the language specified for the code block

Also if the language is not supported, we need to return an empty string, which let's the markdown component know that it should render the original text, that's what the logic above did

File: src/styles/prism.css.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
+/* http://prismjs.com/download.html?themes=prism&languages=markup+css+clike+javascript */
+/**
+ * prism.js default theme for JavaScript, CSS and HTML
+ * Based on dabblet (http://dabblet.com)
+ * @author Lea Verou
+ */
+
+export default `
```
Comment (satya164) [inline src/styles/prism.css.js]: If you use the `css` tag you'll get syntax highlighting and you also won't have to include it manually in `buildCSS` file

**Comment by satya164** [comment]
> Are you willing to bump and publish a new version?

All commits to master are automatically published to npm.

File: src/templates/Markdown.js (hunk 2 above)
Comment (jukben) [reply src/templates/Markdown.js]: Ok, makes sense but what should I do when language is `undefined`?

File: src/styles/prism.css.js (hunk 3 above)
Comment (jukben) [reply src/styles/prism.css.js]: I don't care about highlighting it's copy paste from Prismjs, but I could move it to Markdown.js if you want

File: src/templates/Markdown.js (hunk 1 above)
Comment (jukben) [reply src/templates/Markdown.js]: Yeah, I definitely should

File: src/styles/prism.css.js (hunk 3 above)
Comment (jukben) [reply src/styles/prism.css.js]: Also `illuminate-js` doesn't support the `sh` language, what should I do about it?

File: src/templates/Markdown.js (hunk 2 above)
Comment (jukben) [reply src/templates/Markdown.js]: What about something like this `highlight(text, getLanguage(language) ? language : 'js')`?

File: src/styles/prism.css.js (hunk 3 above)
Comment (satya164) [reply src/styles/prism.css.js]: Does it support `bash`? If it doesn't support a language we should return empty string

**Pull request merged**